src/Model.jl

```diff
@@ -240,3 +246,67 @@ function compute_unit_cell_volume(lattice)
     n_dim = count(!iszero, eachcol(lattice))
     abs(det(lattice[1:n_dim, 1:n_dim]))
 end
+
+
+function Base.show(io::IO, model::Model)
```

Comment:
Move all these functions to printing.jl? Both Model and (especially) pwbasis are pretty crowded already

Reply:
Not sure. The point is that this ought to be changed whenever you change the details of the struct definition (e.g. add a field). So having both in the same file is an advantage. I think we better move other things out.

Reply:
I don't think this point of view (that everything that depends on details of the struct definition belongs to the file) is sustainable. We already have dependencies on "inner details" of pwbasis/model in a number of places, and I think it's completely fine. File splitting is done at a conceptual level of splitting a codebase along logical axes, not at a "java/python" level where it matters to the language

Reply:
Yes, but that stuff breaks in case we change the struct and we catch it in the tests. If you rely on accurate printing in your science and that gets out of sync with the definition that can have pretty bad consequences. I've split out some other things into separate files, where I think it makes more sense.

Reply:
I don't understand your point. Having it in the same file does not make it more or less likely that it stays in sync with the changes in the structure: that's the job of the tests (partially). We already have other places where the changes in X.jl need to be taken into account in Y.jl, that doesn't bother me at all. OTOH printing is kind of orthogonal to all other functionality and so having it all in printing.jl makes sense to me